clang/test/SemaHLSL/BuiltIns/fmod-errors.hlsl

```diff
@@ -0,0 +1,37 @@
+
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected
+
+float test_too_few_arg() {
+  return __builtin_elementwise_fmod();
+  // expected-error@-1 {{too few arguments to function call, expected 2, have 0}}
+}
+
+float2 test_too_many_arg(float2 p0, float2 p1, float2 p3) {
```

Comment:
most of these test cases seem to be covered by `clang/test/Sema/builtins-elementwise-math.c`. You should be able to reduce this test down to just the two double test cases

Reply:
Done. Removed the redundant tests in `clang/test/SemaHLSL/BuiltIns/fmod-errors.hlsl`